R/git.R

```diff
@@ -63,18 +63,52 @@ git_path <- function(git_binary_name = NULL) {
 
 # GitHub ------------------------------------------------------------------
 
-github_info <- function(path = ".") {
+uses_github <- function(path = ".") {
   if (!uses_git(path))
-    return(github_dummy)
+    return(FALSE)
 
   r <- git2r::repository(path, discover = TRUE)
-  if (!("origin" %in% git2r::remotes(r)))
+  r_remote_urls <- remote_urls(r)
+
+  any(grepl("github", r_remote_urls))
+
+  ## or require ALSO that name of matching remote be in github_remote_names?
```

Comment:
ok I removed this comment and revert to plain old git2r::remote_url() since names will not be vetted